lib/registry.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+var protobuf = require('protobufjs');
+var path = require('path');
+// TODO consider the possibility this gets loaded too late
```

Comment:
this is what i was doing in our code before contributing it, but i figured you didn't want the generated code in the repo.

i'm assuming you mean npm build time. now that you say, that makes the most sense. great idea, thanks!